hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/ContextTransactionProcessor.java

```diff
@@ -116,7 +117,11 @@ public CallOutcome call() {
         } catch (AbortException e) {
             // Commit any HAPI fees that were charged before aborting
             rootProxyWorldUpdater.commit();
-            final var result = HederaEvmTransactionResult.fromAborted(e.senderId(), hevmTransaction, e.getStatus());
+            final var result = HederaEvmTransactionResult.fromAborted(
+                    e.senderId(),
+                    hevmTransaction,
+                    e.getStatus(),
+                    context.recordBuilder(ContractOperationRecordBuilder.class).getSignerNonce());
```

Comment:
I think it's safer to re-read the sender account (if present) and call `getNonce()` rather than relying on the `recordBuilder` always having  the right value. 

(What value do we want to externalize if the sender doesn't exist?)